pkg/cli/admin/prune/images/images.go

```diff
@@ -852,8 +852,12 @@ func getRegistryClient(clientConfig *restclient.Config, registryCABundle string,
 		registryCABundleIncluded = true
 	}
 
-	// we have to set a username to something for the Docker login but it's not actually used
-	registryClientConfig.Username = "unused"
+	// Docker client-go library requires only one authentication method to be present. If we
+	// don't have a token file set we need to define at least an username so basic auth will
+	// be the chosen authentication method.
+	if registryClientConfig.BearerTokenFile == "" {
+		registryClientConfig.Username = "unused"
```

Comment:
@ricardomaraschini I guess you want to zero BearerTokenFile in the `// zero out everything we don't want to use` block above.

Reply:
Oh yes, thanks!